src/contrasts.jl

```diff
@@ -598,6 +604,10 @@ end
 StatsAPI.coefnames(C::HypothesisCoding, levels::AbstractVector, baseind::Int) =
     something(C.labels, DataAPI.unwrap.(levels[1:length(levels) .!= baseind]))
 
+# We need an explicit method for `AbstractVector{Bool}` to avoid an ambiguity
+StatsAPI.coefnames(C::HypothesisCoding, levels::AbstractVector{Bool}, baseind::Int) =
+    something(C.labels, DataAPI.unwrap.(levels[1:length(levels) .!= baseind]))
```

Comment:
I wonder whether it's worth using `invoke` to avoid duplicating the method body?
```suggestion
    invoke(coefnames, Tuple{HypothesisCoding,AbstractVector,Int}, C, levels, baseind)
```
Probably doesn't really matter.

Reply:
Hmmm, I'll consider that next time, going with the "no change" lazy and fast option for now 😂